Projects/My_website/myWebsite/blog/urls.py

```diff
@@ -7,8 +7,7 @@
 router = routers.DefaultRouter()
 router.register("all-blogs", views.blog)
 router.register("my-blogs", views.myBlog)
+router.register("blog-comments", views.blogComments)
+router.register("add-blog-comment", views.addComment)
```

Comment:
@codal-uchoudhary Please avoid using keywords like add, update, etc. in Enpoint.